### ISSUE TITLE
"addmm_impl_cpu_" not implemented for 'Half'

### ISSUE THREAD

**Issue by ch1ld3r1c0** [huggingface/transformers#29465]
### System Info

- `transformers` version: 4.38.2
- Platform: Linux-5.14.0-284.11.1.el9_2.x86_64-x86_64-with-glibc2.34
- Python version: 3.9.14
- Huggingface_hub version: 0.21.3
- Safetensors version: 0.4.2
- Accelerate version: 0.27.2
- Accelerate config:    not found
- PyTorch version (GPU?): 1.13.1+cu117 (False)
- Tensorflow version (GPU?): not installed (NA)
- Flax version (CPU?/GPU?/TPU?): not installed (NA)
- Jax version: not installed
- JaxLib version: not installed
- Using GPU in script?: <fill in>
- Using distributed or parallel set-up in script?: <fill in>

### Information

- [X] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [X] My own task or dataset (give details below)

### Reproduction

from spacy_llm.util import assemble
nlp = assemble("spacy_few_shots.cfg")

text="""
example tex
"""

doc = nlp(text)
for e in doc.ents:
    print(e.label_, e.text, e.start, e.end)

_______________________________
spacy_few_shots.cfg
[nlp]
lang = "en"
pipeline = ["llm"]

[components]

[components.llm]
factory = "llm"

[components.llm.task]
@llm_tasks = "spacy.NER.v2"
labels = label(example)

[components.llm.task.examples]
@misc = "spacy.FewShotReader.v1"
path = "spacy_few_shots.yml"

[components.llm.model]
@llm_models = "spacy.OpenLLaMA.v1"
name = "open_llama_3b"

### Expected behavior

Model should generate output.

**Comment by erfanzar** [comment]
The issue you are facing right now is not from the hugging face/transformers team and that's happening cause you are using `model = model.half()` in CPU, and matmul op for CPU is not implemented in PyTorch

**Comment by ArthurZucker** [comment]
Closing as @erfanzar's answer resolves this issue
